dist/testbed-support/Makefile.iotlab

```diff
@@ -0,0 +1,29 @@
+.PHONY: iotlab-auth iotlab-exp iotlab-flash iotlab-reset iotlab-term
+
+IOTLAB_NODES    ?= 5
+IOTLAB_DURATION ?= 30
+IOTLAB_SITE     ?= grenoble
+IOTLAB_TYPE     ?= "m3:at86rf231"
+IOTLAB_AUTH     ?= $(HOME)/.iotlabrc
+IOTLAB_USER     ?= $(shell cut -f1 -d: $(IOTLAB_AUTH))
+IOTLAB_EXP_ID   ?= $(shell experiment-cli get -l --state Running | grep -m 1 '"id"' | grep -Eo '[[:digit:]]+')
+
+$(IOTLAB_AUTH):
+	auth-cli -u $(IOTLAB_USER)
+
+iotlab-exp: $(IOTLAB_AUTH)
+    ifeq (,$(AD))
```

Comment:
Tab vs. Spaces. I never understand when Make requires which of them, but with tabs here it didn't work properly.